src/presets.js

```diff
@@ -44,19 +44,20 @@ const templateDefaults = [
 	'{{> block-newline}}',
 	'{{#each commits}}',
 		'{{> render-with-nesting nesting=../nesting block=../block ~}}',
-	'{{/each~}}',
+	'{{/each}}',
+	'{{> block-newline}}',
```

Comment:
This just adds more separation between commits so you get
```
>> ### module-13.0.2
>> #### (2020-06-29)
>>
>> * Commits
>>
>> ### module-13.0.1
>> #### (2020-06-22)
```
instead of
```
>> ### module-13.0.2
>> #### (2020-06-29)
>>
>> * Commits
>> ### module-13.0.1
>> #### (2020-06-22)
```
The actual rendering is identical but the extra spacing helps a lot when looking at the markdown directly